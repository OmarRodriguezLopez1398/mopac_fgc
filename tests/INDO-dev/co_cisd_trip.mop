INDO CISD C.I.=(6,3) C.I.D.=(3,1) TRIPLET 
Carbon Monoxide
INDO 
C  0.000000 0   0.000000 0   0.000000 0
O  1.173000 0   0.000000 0   0.000000 0
