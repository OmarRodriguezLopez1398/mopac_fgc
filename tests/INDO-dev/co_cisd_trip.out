 *******************************************************************************
 **                                                                           **
 **                              MOPAC v22.0.6                                **
 **                                                                           **
 *******************************************************************************
 **          Digital Object Identifier (DOI): 10.5281/zenodo.6511958          **
 **    Visit the DOI location for information on how to cite this program.    **
 *******************************************************************************

                              INDO CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon May 15 10:16:30 2023  *
 *  INDO       - The INDO Hamiltonian to be used
 *  CISD       - C.I. USES 1 AND 2 ELECTRON EXCITATIONS
 *  TRIPLET    - SPIN STATE DEFINED AS A TRIPLET
 * C.I.D.=(N,M)- 1 DOUBLY FILLED LEVELS USED IN A CI DOUBLES INVOLVING 3 M.O.'S
 *  C.I.=(N,M) - 3 DOUBLY FILLED LEVELS USED IN A C.I. INVOLVING 6 M.O.'S
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
INDO CISD C.I.=(6,3) C.I.D.=(3,1) TRIPLET
 Carbon Monoxide
 INDO
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)

     1       C          0.00000000      0.00000000      0.00000000   
     2       O          1.17300000      0.00000000      0.00000000   



          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         C          0.0000      0.0000      0.0000
     2         O          1.1730      0.0000      0.0000

           Empirical Formula: C O  =     2 atoms



      MOLECULAR POINT GROUP   :   C*v 


 TRIPLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 5

 WARNING: INDO only performs single-point calculations

 Summary of symmetry occupancy in each shell:

 SHELL     
    1    5
    v    3
 TOTAL   8


                     MOLECULAR ORBITALS 






    ROOT NO.    1           2           3           4           5           6

           -43.05652   -19.27644   -14.77940   -14.77940   -14.49534     0.09603
  
  
  S  C   1   0.44267    -0.58677    -0.00000     0.00000    -0.60295     0.00000
 PX  C   1   0.36283     0.01825     0.00000     0.00000     0.61061    -0.00000
 PY  C   1  -0.00000    -0.00000     0.48812    -0.00000    -0.00000     0.87278
 PZ  C   1   0.00000    -0.00000    -0.00000     0.48812     0.00000     0.00000
  
  S  O   2   0.77781     0.52976    -0.00000     0.00000    -0.10914     0.00000
 PX  O   2  -0.25962     0.61215    -0.00000     0.00000    -0.50170     0.00000
 PY  O   2   0.00000     0.00000     0.87278     0.00000    -0.00000    -0.48812
 PZ  O   2  -0.00000     0.00000    -0.00000     0.87278     0.00000    -0.00000




    ROOT NO.    7           8

             0.09603    10.49950
  
  
  S  C   1   0.00000     0.31016
 PX  C   1   0.00000     0.70369
 PY  C   1  -0.00000     0.00000
 PZ  C   1   0.87278    -0.00000
  
  S  O   2   0.00000    -0.32009
 PX  O   2   0.00000     0.55333
 PY  O   2  -0.00000    -0.00000
 PZ  O   2  -0.48812    -0.00000


 Reference determinate nber   1 is ALPHA 1 -   5
 SINGLE excitations FROM orbs   3 to   5 INTO orbs   6 to   8
 DOUBLE excitations FROM orbs   5 to   5 INTO orbs   6 to   7


 CI excitations=   34:        = 34
 The lowest   34 spin-adapted configurations of multiplicity=  3
     sym   eV   cm**-1 -dets- dipole oscilator X FRAG ....Excitations named from first reference determinate
                       tot  #  Debye  strength

   1      5.184      0.  2  1  1.8085 0.000000 1  1 (   5    )->(   6       )
   2      5.184      0.  2  1  1.8085 0.000000 1  1 (   5    )->(   7       )
   3      5.940   6101.  2  1  2.4836 0.001920 1  1 (   3    )->(   6       )
   4      5.940   6101.  2  1  2.4836 0.000000 1  1 (   4    )->(   7       )
   5      6.437  10111.  2  1  2.4836 0.000000 1  1 (   3    )->(   7       )
   6      6.437  10111.  2  1  2.4836 0.003181 1  1 (   4    )->(   6       )
   7     12.035  55262.  6  1  1.1407 0.000000 2  1 (   3   5)->(   6   7   )
   8     12.035  55262.  6  1  1.1407 0.001436 2  1 (   4   5)->(   6   7   )
   9     13.068  63589.  2  1  1.1407 0.000000 2  1 (   4   5)->(   6   6   )
  10     13.068  63589.  2  1  1.1407 0.000000 2  1 (   3   5)->(   7   7   )
  11     13.210  64741.  2  1  3.1514 0.036976 2  1 (   5   5)->(   6   7   )
  12     13.246  65029.  6  2  1.1407 0.000000 2  1 (   3   5)->(   6   7   )
  13     13.246  65029.  6  2  1.1407 0.004012 2  1 (   4   5)->(   6   7   )
  14     14.126  72122.  2  1  1.1407 0.195364 2  1 (   3   5)->(   6   6   )
  15     14.126  72122.  2  1  1.1407 0.000000 2  1 (   4   5)->(   7   7   )
  16     15.991  87164.  6  3  1.1407 0.000000 2  1 (   3   5)->(   6   7   )
  17     15.991  87164.  6  3  1.1407 0.464576 2  1 (   4   5)->(   6   7   )
  18     16.203  88878.  2  1  1.1095 0.000000 1  1 (   3    )->(   8       )
  19     16.203  88878.  2  1  1.1095 0.000000 1  1 (   4    )->(   8       )
  20     16.403  90487.  2  1  3.1826 0.031694 1  1 (   5    )->(   8       )
  21     22.662 140969.  6  1  0.2334 0.000000 2  1 (   3   5)->(   7   8   )
  22     22.662 140969.  6  1  0.2334 0.000550 2  1 (   4   5)->(   6   8   )
  23     22.956 143342.  6  1  0.2334 0.000977 2  1 (   3   5)->(   6   8   )
  24     22.956 143342.  6  1  0.2334 0.000000 2  1 (   4   5)->(   7   8   )
  25     23.509 147807.  6  2  0.2334 0.000000 2  1 (   3   5)->(   7   8   )
  26     23.509 147807.  6  2  0.2334 0.000717 2  1 (   4   5)->(   6   8   )
  27     23.792 150085.  6  2  0.2334 0.000000 2  1 (   4   5)->(   7   8   )
  28     23.792 150085.  6  2  0.2334 0.000233 2  1 (   3   5)->(   6   8   )
  29     24.366 154715.  6  3  0.2334 0.000000 2  1 (   3   5)->(   7   8   )
  30     24.366 154715.  6  3  0.2334 0.000365 2  1 (   4   5)->(   6   8   )
  31     24.989 159742.  2  1  4.5255 0.226336 2  1 (   5   5)->(   6   8   )
  32     24.989 159742.  2  1  4.5255 0.000000 2  1 (   5   5)->(   7   8   )
  33     26.402 171139.  6  3  0.2334 0.000000 2  1 (   4   5)->(   7   8   )
  34     26.402 171139.  6  3  0.2334 0.000469 2  1 (   3   5)->(   6   8   )

 Depression of ground-state after CI=      41212. cm**-1 Energy=  -457.2417030 eV
 Dipole moment= -2.082719 -0.000000 -0.000000 tot=  2.082719 Debyes


  CI trans.  energy frequency wavelength oscillator--------- polarization---------  dipole  ------ components-----
 st.  symm.    eV      cm - 1       nm      strength       x          y         z       moment     x       y       z 

   2      0.0000000        0.      0.00  0.000000                                   2.082719  -2.083  -0.000   0.000
   3      0.4061393     3276.   3052.74  0.000437  -0.000000   0.037937   0.999280  1.984448   1.984   0.000  -0.000
   4      0.9032850     7286.   1372.59  0.000842   0.000000  -0.037937   0.999280  1.984260   1.984   0.000   0.000
   5      0.9032850     7286.   1372.59  0.000842  -0.000000   0.999280   0.037937  1.984260   1.984   0.000   0.000
   6      1.3665161    11022.    907.30  0.001003  -0.000000   0.999280  -0.037937  1.957653   1.958   0.000   0.000
   7      6.6719357    53813.    185.83  0.001341  -1.000000  -0.000000  -0.000000  1.124355   1.124  -0.000  -0.000
   8      6.6719357    53813.    185.83  0.000003   1.000000   0.000000   0.000000  1.124355   1.124   0.000   0.000
   9      7.3484662    59270.    168.72  0.000000                                   1.140678   1.141   0.000   0.000
  10      7.3484662    59270.    168.72  0.000000                                   1.140678   1.141  -0.000  -0.000
  11      8.1345469    65610.    152.42  0.034796  -0.000000  -0.999280   0.037937  3.124224  -3.124  -0.000   0.000
  12      8.1455097    65698.    152.21  0.000392  -1.000000   0.000000  -0.000000  1.116427   1.116   0.000   0.000
  13      8.1455097    65698.    152.21  0.003207  -1.000000   0.000000  -0.000000  1.116427   1.116   0.000  -0.000
  14      8.2877862    66846.    149.60  0.000002   1.000000  -0.000000   0.000000  1.143750   1.144  -0.000   0.000
  15      8.2877862    66846.    149.60  0.001046   1.000000  -0.000000   0.000000  1.143750   1.144   0.000  -0.000
  16     11.0979448    89511.    111.72  0.000002   1.000000   0.000000   0.000000  1.101988   1.102  -0.000  -0.000
  17     11.0979448    89511.    111.72  0.001362   1.000000   0.000000  -0.000000  1.101988   1.102   0.000   0.000
  18     11.1786409    90162.    110.91  0.032505  -0.000000  -0.037937  -0.999280  2.454143  -2.454   0.000  -0.000
  19     12.1294657    97831.    102.22  0.009818  -1.000000   0.000000  -0.000000  1.236018   1.236   0.000  -0.000
  20     12.1294657    97831.    102.22  0.559141  -1.000000   0.000000   0.000000  1.236018   1.236   0.000   0.000
  21     17.3332792   139803.     71.53  0.000049  -0.000000   0.999280  -0.037937  0.227487  -0.227   0.000  -0.000
  22     17.6216760   142129.     70.36  0.000757  -0.000000   0.037937   0.999280  0.252290  -0.252  -0.000   0.000
  23     17.7369258   143058.     69.90  0.000072  -0.000000  -0.999113  -0.042109  0.222521  -0.223  -0.000   0.000
  24     17.7369258   143058.     69.90  0.000072   0.000000   0.042109  -0.999113  0.222521  -0.223  -0.000  -0.000
  25     18.2019261   146809.     68.12  0.000437  -0.000000   0.999278   0.037991  0.221749  -0.222   0.000  -0.000
  26     18.2019261   146809.     68.12  0.000437   0.000000  -0.037991   0.999278  0.221749  -0.222   0.000  -0.000
  27     18.6320550   150278.     66.54  0.000273  -0.000000  -0.999280   0.037937  0.233063  -0.233  -0.000   0.000
  28     18.6580266   150488.     66.45  0.001985   0.000000   0.037937   0.999280  0.228422   0.228   0.000   0.000
  29     19.1961789   154828.     64.59  0.001336   0.000000  -0.997933  -0.064262  0.243424   0.243   0.000  -0.000
  30     19.1961789   154828.     64.59  0.001336  -0.000000  -0.064262   0.997933  0.243424   0.243   0.000  -0.000
  31     19.6669850   158625.     63.04  0.001360   0.000000  -0.999280   0.037937  0.259119   0.259  -0.000  -0.000
  32     20.2831547   163595.     61.13  0.482377   1.000000  -0.000000   0.000000  4.301606  -4.302   0.000   0.000
  33     20.2831547   163595.     61.13  0.000390  -1.000000  -0.000000   0.000000  4.301606  -4.302   0.000   0.000
  34     24.6927578   199161.     50.21  0.000219  -0.000000   0.037937   0.999280  0.905610  -0.906   0.000  -0.000

 Polarizability (au  ) xx=   11.44 xy=   -0.00 yy=    4.68 xz=   -0.00 yz=    0.16 zz=    8.79
 Polarizability (A**3) xx=    1.70 xy=   -0.00 yy=    0.69 xz=   -0.00 yz=    0.02 zz=    1.30


 Major CI contributions to CI states

State    1  0.0000     CI coeff  CI percent
    Config       2  -0.99508029  0.99018479
    Config       7  -0.04614030  0.00212893
    Config      15   0.06337219  0.00401604
 Total coeff printed             0.99632976
  
State    2  0.0000     CI coeff  CI percent
    Config       1   0.99508029  0.99018479
    Config       8   0.04614030  0.00212893
    Config      14  -0.06337219  0.00401604
 Total coeff printed             0.99632976
  
State    3  0.4061     CI coeff  CI percent
    Config       3  -0.70369290  0.49518370
    Config       4   0.70369290  0.49518370
    Config      27   0.06881566  0.00473559
    Config      28   0.06881566  0.00473559
 Total coeff printed             0.99983858
  
State    4  0.9033     CI coeff  CI percent
    Config       3   0.70369100  0.49518102
    Config       4   0.70369100  0.49518102
    Config      27   0.06883383  0.00473810
    Config      28  -0.06883383  0.00473810
 Total coeff printed             0.99983824
  
State    5  0.9033     CI coeff  CI percent
    Config       5  -0.70369100  0.49518102
    Config       6  -0.70369100  0.49518102
    Config      29  -0.06845649  0.00468629
    Config      30  -0.06845649  0.00468629
 Total coeff printed             0.99973463
  
State    6  1.3665     CI coeff  CI percent
    Config       5  -0.70191605  0.49268614
    Config       6   0.70191605  0.49268614
    Config      11  -0.07075773  0.00500666
    Config      29  -0.06840084  0.00467868
    Config      30   0.06840084  0.00467868
 Total coeff printed             0.99973628
  
State    7  6.6719     CI coeff  CI percent
    Config       2  -0.06207833  0.00385372
    Config       7   0.80244381  0.64391606
    Config       9   0.22746319  0.05173950
    Config      12   0.05657702  0.00320096
    Config      15  -0.46527139  0.21647747
    Config      16   0.28395964  0.08063308
 Total coeff printed             0.99982079
  
State    8  6.6719     CI coeff  CI percent
    Config       1  -0.06207833  0.00385372
    Config       8   0.80244381  0.64391606
    Config      10   0.22746319  0.05173950
    Config      13   0.05657702  0.00320096
    Config      14  -0.46527139  0.21647747
    Config      17   0.28395964  0.08063308
 Total coeff printed             0.99982079
  
State    9  7.3485     CI coeff  CI percent
    Config       7   0.57511525  0.33075755
    Config       8  -0.04503805  0.00202843
    Config       9  -0.49847385  0.24847618
    Config      12  -0.14891205  0.02217480
    Config      15   0.49847385  0.24847618
    Config      16  -0.37949969  0.14402001
 Total coeff printed             0.99593314
  
State   10  7.3485     CI coeff  CI percent
    Config       7  -0.04503805  0.00202843
    Config       8  -0.57511525  0.33075755
    Config      10   0.49847385  0.24847618
    Config      13   0.14891205  0.02217480
    Config      14  -0.49847385  0.24847618
    Config      17   0.37949969  0.14402001
 Total coeff printed             0.99593314
  
State   11  8.1345     CI coeff  CI percent
    Config       5   0.05004226  0.00250423
    Config       6  -0.05004226  0.00250423
    Config      11  -0.99748601  0.99497834
 Total coeff printed             0.99998680
  
State   12  8.1455     CI coeff  CI percent
    Config       9  -0.14722282  0.02167456
    Config      10  -0.37465497  0.14036634
    Config      12   0.33241767  0.11050151
    Config      13   0.84594177  0.71561749
    Config      14  -0.08030162  0.00644835
 Total coeff printed             0.99460825
  
State   13  8.1455     CI coeff  CI percent
    Config       9  -0.37465497  0.14036634
    Config      10   0.14722282  0.02167456
    Config      12   0.84594177  0.71561749
    Config      13  -0.33241767  0.11050151
    Config      15  -0.08030162  0.00644835
 Total coeff printed             0.99460825
  
State   14  8.2878     CI coeff  CI percent
    Config       8  -0.13747489  0.01889935
    Config       9  -0.05734326  0.00328825
    Config      10  -0.72701182  0.52854619
    Config      13  -0.38143628  0.14549363
    Config      14  -0.44977916  0.20230129
    Config      17   0.30548058  0.09331838
    Config      18   0.07163295  0.00513128
 Total coeff printed             0.99697837
  
State   15  8.2878     CI coeff  CI percent
    Config       7  -0.13747489  0.01889935
    Config       9  -0.72701182  0.52854619
    Config      10   0.05734326  0.00328825
    Config      12  -0.38143628  0.14549363
    Config      15  -0.44977916  0.20230129
    Config      16   0.30548058  0.09331838
    Config      19  -0.07163295  0.00513128
 Total coeff printed             0.99697837
  
State   16 11.0979     CI coeff  CI percent
    Config      10  -0.06310595  0.00398236
    Config      17   0.04986942  0.00248696
    Config      18  -0.99594444  0.99190533
 Total coeff printed             0.99837465
  
State   17 11.0979     CI coeff  CI percent
    Config       9   0.06310595  0.00398236
    Config      16  -0.04986942  0.00248696
    Config      19  -0.99594444  0.99190533
 Total coeff printed             0.99837465
  
State   18 11.1786     CI coeff  CI percent
    Config      20   0.99406659  0.98816839
    Config      23  -0.04001737  0.00160139
    Config      24  -0.04001737  0.00160139
    Config      27  -0.04847960  0.00235027
    Config      28  -0.04847960  0.00235027
    Config      33   0.04419600  0.00195329
    Config      34   0.04419600  0.00195329
 Total coeff printed             0.99997828
  
State   19 12.1295     CI coeff  CI percent
    Config       1  -0.05011168  0.00251118
    Config       8  -0.04079868  0.00166453
    Config      14  -0.54775661  0.30003731
    Config      15   0.05152902  0.00265524
    Config      16   0.07523338  0.00566006
    Config      17  -0.79973541  0.63957673
    Config      31   0.21568373  0.04651947
 Total coeff printed             0.99862452
  
State   20 12.1295     CI coeff  CI percent
    Config       2   0.05011168  0.00251118
    Config       7   0.04079868  0.00166453
    Config      14   0.05152902  0.00265524
    Config      15   0.54775661  0.30003731
    Config      16   0.79973541  0.63957673
    Config      17   0.07523338  0.00566006
    Config      32   0.21568373  0.04651947
 Total coeff printed             0.99862452
  
State   21 17.3333     CI coeff  CI percent
    Config      21  -0.70382863  0.49537474
    Config      22   0.70382863  0.49537474
    Config      25  -0.05833770  0.00340329
    Config      26   0.05833770  0.00340329
 Total coeff printed             0.99755606
  
State   22 17.6217     CI coeff  CI percent
    Config      20   0.05138612  0.00264053
    Config      23   0.70497758  0.49699339
    Config      24   0.70497758  0.49699339
 Total coeff printed             0.99662732
  
State   23 17.7369     CI coeff  CI percent
    Config      21   0.68647464  0.47124743
    Config      22   0.68647464  0.47124743
    Config      25  -0.16736255  0.02801022
    Config      26  -0.16736255  0.02801022
 Total coeff printed             0.99851531
  
State   24 17.7369     CI coeff  CI percent
    Config      23   0.36148832  0.13067381
    Config      24  -0.36148832  0.13067381
    Config      27  -0.17573133  0.03088150
    Config      28   0.17573133  0.03088150
    Config      33   0.58161198  0.33827250
    Config      34  -0.58161198  0.33827250
 Total coeff printed             0.99965561
  
State   25 18.2019     CI coeff  CI percent
    Config      21   0.16670264  0.02778977
    Config      22   0.16670264  0.02778977
    Config      25   0.68689783  0.47182863
    Config      26   0.68689783  0.47182863
 Total coeff printed             0.99923679
  
State   26 18.2019     CI coeff  CI percent
    Config      23  -0.60666846  0.36804662
    Config      24   0.60666846  0.36804662
    Config      27  -0.14415367  0.02078028
    Config      28   0.14415367  0.02078028
    Config      33   0.33335887  0.11112813
    Config      34  -0.33335887  0.11112813
 Total coeff printed             0.99991007
  
State   27 18.6321     CI coeff  CI percent
    Config      21   0.05892245  0.00347186
    Config      22  -0.05892245  0.00347186
    Config      25  -0.70456447  0.49641109
    Config      26   0.70456447  0.49641109
 Total coeff printed             0.99976590
  
State   28 18.6580     CI coeff  CI percent
    Config       3  -0.06923736  0.00479381
    Config       4   0.06923736  0.00479381
    Config      20  -0.06664829  0.00444199
    Config      27  -0.69979957  0.48971944
    Config      28  -0.69979957  0.48971944
    Config      33  -0.04934814  0.00243524
    Config      34  -0.04934814  0.00243524
 Total coeff printed             0.99833898
  
State   29 19.1962     CI coeff  CI percent
    Config       5   0.06787730  0.00460733
    Config       6   0.06787730  0.00460733
    Config      29  -0.70289646  0.49406344
    Config      30  -0.70289646  0.49406344
 Total coeff printed             0.99734153
  
State   30 19.1962     CI coeff  CI percent
    Config       3   0.06787730  0.00460733
    Config       4   0.06787730  0.00460733
    Config      27  -0.66580132  0.44329140
    Config      28   0.66580132  0.44329140
    Config      33  -0.22471143  0.05049523
    Config      34   0.22471143  0.05049523
 Total coeff printed             0.99678791
  
State   31 19.6670     CI coeff  CI percent
    Config       5   0.06887201  0.00474335
    Config       6  -0.06887201  0.00474335
    Config      29  -0.70288272  0.49404412
    Config      30   0.70288272  0.49404412
 Total coeff printed             0.99757495
  
State   32 20.2832     CI coeff  CI percent
    Config      15  -0.12318802  0.01517529
    Config      16  -0.17672193  0.03123064
    Config      31   0.06469220  0.00418508
    Config      32   0.97322413  0.94716522
 Total coeff printed             0.99775622
  
State   33 20.2832     CI coeff  CI percent
    Config      14   0.12318802  0.01517529
    Config      17   0.17672193  0.03123064
    Config      31   0.97322413  0.94716522
    Config      32  -0.06469220  0.00418508
 Total coeff printed             0.99775622
  
State   34 24.6928     CI coeff  CI percent
    Config      20  -0.06889299  0.00474624
    Config      27  -0.04567243  0.00208597
    Config      28  -0.04567243  0.00208597
    Config      33   0.70354374  0.49497380
    Config      34   0.70354374  0.49497380
 Total coeff printed             0.99886579
  

 -------------------------------------------------------------------------------
 INDO CISD C.I.=(6,3) C.I.D.=(3,1) TRIPLET
 Carbon Monoxide
 INDO


     NO PARAMETERS MARKED FOR OPTIMIZATION, SO 1SCF WAS USED  
     SCF FIELD WAS ACHIEVED                                   


                              INDO CALCULATION
                                                       MOPAC v22.0.6 MacOS
                                                       Mon May 15 10:16:30 2023


          COSMO AREA              =         57.92 SQUARE ANGSTROMS
          COSMO VOLUME            =         40.40 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =         14.495342 EV
          HOMO LUMO ENERGIES (EV) =        -14.495  0.096
          NO. OF FILLED LEVELS    =          5
          MOLECULAR WEIGHT        =         28.0104         POINT GROUP:  C*v 

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            O     2    C     1     1.17300
            O     2    C     1     0.00000
            C     1    O     2     0.00000


          SCF CALCULATIONS        =          1

          WALL-CLOCK TIME         =      0.059 SECONDS
          COMPUTATION TIME        =      0.057 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)

     1       C          0.00000000      0.00000000      0.00000000   
     2       O          1.17300000      0.00000000      0.00000000   

                             CARTESIAN COORDINATES

   1    C        0.000000000     0.000000000     0.000000000
   2    O        1.173000000     0.000000000     0.000000000


           Empirical Formula: C O  =     2 atoms



      MOLECULAR POINT GROUP   :   C*v 


                  EIGENVALUES  
 -43.05652 -19.27644 -14.77940 -14.77940 -14.49534   0.09603   0.09603  10.49950


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C           0.228558        3.7714     1.80762     1.96383
    2          O          -0.228558        6.2286     1.79506     4.43350
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.    -1.288     0.000     0.000     1.288
 HYBRID         0.828    -0.000    -0.000     0.828
 SUM           -0.460    -0.000    -0.000     0.460


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.80762   1.00993   0.47695   0.47695
    2  O   1.79506   1.38739   1.52305   1.52305

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.10 SECONDS

 == MOPAC DONE ==
